backend/apiServer/controller/repo.go

```diff
@@ -88,3 +90,87 @@ func (repo RepoController) NewRepoFromURI(w http.ResponseWriter, r *http.Request
 
 	return
 }
+
+
+/**
+* @api {Post} /repo/:id Parse the repository assosiated with id.
+* @apiName Parse repository.
+* @apiGroup Repository
+* @apiPermission none
+*
+* @apiParam {String} Id Id of submitted git repository.
+*
+* @apiParamExample {url} Parse repository:
+* 	/repo/:id
```

Comment:
@apiParamExample should be an example input (http://apidocjs.com/#param-api-param-example).
/repo/:id is given by @apiParam.